micro-ui/web/micro-ui-internals/packages/modules/sandbox/src/pages/employee/ProductDetails/ProductDetailsComponentUpdated.js

```
@@ -0,0 +1,484 @@
+(
+    <div style={{ width: '100%', backgroundColor: '#ffffff', padding: '0rem 2rem 0rem 4rem' }}>
+        <div className="about-container">
+            <div className="about-title-wrapper">
+                <h2 className="about-title">{about.title}</h2>
+                <div className="title-underline"></div>
+            </div>
+            {about.paragraphs.map((para, i) => {
+                const decodedpara = he.decode(para); // decode &lt;strong&gt;
+
+                return (
+                    <p key={i} className="about-description" dangerouslySetInnerHTML={{ __html: decodedpara }}></p>
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)